cytoscape-edgehandles.js

```diff
@@ -449,6 +449,9 @@ SOFTWARE.
       },
       stop: function( sourceNode ) {
         // fired when edgehandles interaction is stopped (either complete with added edges or incomplete)
+      }, 
+      cancel: function( sourceNode, position ) {
```

Comment:
Is this position the rendered (on-screen) position?  Or is it the model position?  If it's the rendered position, we need to say so, because that changes with pan and zoom.